qa/tasks/cephfs/filesystem.py

```diff
@@ -629,13 +634,22 @@ def create(self):
         else:
             data_pool_name = self.data_pool_name
 
+        # will use the ec pool to store the data and a small amount of
+        # metadata still goes to the primary data pool for all files.
+        if not self.metadata_overlay and self.ec_profile and 'disabled' not in self.ec_profile:
+            self.target_size_ratio = 0.05
+
         log.debug("Creating filesystem '{0}'".format(self.name))
 
         self.mon_manager.raw_cluster_cmd('osd', 'pool', 'create',
-                                         self.metadata_pool_name, self.pgs_per_fs_pool.__str__())
+                                         self.metadata_pool_name,
+                                         '--pg_num_min', str(self.pg_num_min))
 
         self.mon_manager.raw_cluster_cmd('osd', 'pool', 'create',
-                                         data_pool_name, self.pgs_per_fs_pool.__str__())
+                                         data_pool_name, str(self.pg_num),
+                                         '--pg_num_min', str(self.pg_num_min),
+                                         '--target_size_ratio',
+                                         str(self.target_size_ratio))
```

Comment:
It seems the PGs are not clean yet before the test starts going. Try adding

`self.mon_manager.wait_for_clean()`

To see if that helps.

Reply:
Sure, will try it. Thanks.

Reply:
I thought for the whole mds full test it will create the cluster once. Checked the tethology logs found it will recreate the cluster for each test. I will fix and test it.